doc/getting-started/rt_industry_ubuntu.rst

```diff
@@ -0,0 +1,604 @@
+
+      }
+
+   .. note::
+
+      Update this to use your uuid and PARTUUID for the root= parameter (or
+      use the device node directly).
+
+      Update the kernel name if you used a different name as the source
+      for your Service VM kernel.
+
+#. Modify the ``/etc/default/grub`` file to make the grub menu visible when
+   booting and make it load the Service VM kernel by default. Modify the
+   lines shown below:
+
+   .. code-block:: none
+
+      GRUB_DEFAULT=ubuntu-user-vm
+      #GRUB_TIMEOUT_STYLE=hidden
+      GRUB_TIMEOUT=5
+
+#. Update Grub on your system:
+
+   .. code-block:: none
+
+      sudo update-grub
+
+Recommended BIOS settings for RTVM
+----------------------------------
+
+.. csv-table::
+   :widths: 15, 30, 10
+
+   "Hyper-Threading", "Intel Advanced Menu -> CPU Configuration", "Disabled"
+   "Intel VMX", "Intel Advanced Menu -> CPU Configuration", "Enable"
+   "Speed Step", "Intel Advanced Menu -> Power & Performance -> CPU - Power Management Control", "Disabled"
+   "Speed Shift", "Intel Advanced Menu -> Power & Performance -> CPU - Power Management Control", "Disabled"
+   "C States", "Intel Advanced Menu -> Power & Performance -> CPU - Power Management Control", "Disabled"
+   "RC6", "Intel Advanced Menu -> Power & Performance -> GT - Power Management", "Disabled"
+   "GT freq", "Intel Advanced Menu -> Power & Performance -> GT - Power Management", "Lowest"
+   "SA GV", "Intel Advanced Menu -> Memory Configuration", "Fixed High"
+   "VT-d", "Intel Advanced Menu -> System Agent Configuration", "Enable"
+   "Gfx Low Power Mode", "Intel Advanced Menu -> System Agent Configuration -> Graphics Configuration", "Disabled"
+   "DMI spine clock gating", "Intel Advanced Menu -> System Agent Configuration -> DMI/OPI Configuration", "Disabled"
+   "PCH Cross Throttling", "Intel Advanced Menu -> PCH-IO Configuration", "Disabled"
+   "Legacy IO Low Latency", "Intel Advanced Menu -> PCH-IO Configuration -> PCI Express Configuration", "Enabled"
+   "PCI Express Clock Gating", "Intel Advanced Menu -> PCH-IO Configuration -> PCI Express Configuration", "Disabled"
+   "Delay Enable DMI ASPM", "Intel Advanced Menu -> PCH-IO Configuration -> PCI Express Configuration", "Disabled"
+   "DMI Link ASPM", "Intel Advanced Menu -> PCH-IO Configuration -> PCI Express Configuration", "Disabled"
+   "Aggressive LPM Support", "Intel Advanced Menu -> PCH-IO Configuration -> SATA And RST Configuration", "Disabled"
+   "USB Periodic Smi", "Intel Advanced Menu -> LEGACY USB Configuration", "Disabled"
+   "ACPI S3 Support", "Intel Advanced Menu -> ACPI Settings", "Disabled"
+   "Native ASPM", "Intel Advanced Menu -> ACPI Settings", "Disabled"
+
+.. note:: BIOS settings depend on the platform and BIOS version; some may
+   not be applicable.
+
+Recommended kernel cmdline for RTVM
+-----------------------------------
+
+.. code-block:: none
+
+   root=root=PARTUUID=<UUID of rootfs partition> rw rootwait nohpet console=hvc0 console=ttyS0 \
```

Comment:
```suggestion
   root=PARTUUID=<UUID of rootfs partition> rw rootwait nohpet console=hvc0 console=ttyS0 \
```

Reply:
Yes.
@deb-intel Please helps on this.
Thanks!